packages/slider/src/useSlide.ts

```diff
@@ -0,0 +1,111 @@
+import { computed, inject, nextTick, ref,Ref, ComputedRef } from 'vue'
+import { ISliderButton, ISliderInitData, ISliderProps, Slide } from './Slider'
+
+export const useSlide = (props:ISliderProps, initData: ISliderInitData, emit):Slide => {
+  const elForm:{ disabled?: boolean;} = inject('elForm', {})
+
+  const slider = ref(null)
+
+  const button1 = ref(null)
```

Comment:
you should make sense of these vars. Donot use `button1` or `button2`, it's  confusing.